### PR TITLE
feat: Do not look for encryption when flag is not set

### PR DESCRIPTION
**Pull request by paultranvan**
The query is useless if the encryption flag is not set

### CODE REVIEW

File: src/drive/lib/encryption.js [hunk 1]
```diff
@@ -17,6 +18,9 @@ export const isEncryptedFile = file => {
 }
 
 export const getEncryptionKeyFromDirId = async (client, dirId) => {
+  if (flag('drive.enable-encryption')) {
```
Comment (trollepierre) [inline src/drive/lib/encryption.js:21]: I think we should use the opposite
`if (!flag('drive.enable-encryption')) {`

File: src/drive/lib/encryption.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import get from 'lodash/get'
+import flag from 'cozy-flags'
```
Comment (trollepierre) [inline src/drive/lib/encryption.js:2]: - please add the flag  in the list of the development flag: https://github.com/cozy/cozy-drive/blob/master/src/lib/flags.js
- enable the feature in prod' for cozy internal here: https://bender.cozycloud.cc/contexts/prod/cozy_internal

File: src/drive/lib/encryption.js [hunk 3]
```diff
@@ -17,6 +18,9 @@ export const isEncryptedFile = file => {
 }
 
 export const getEncryptionKeyFromDirId = async (client, dirId) => {
+  if (flag('drive.enable-encryption')) {
+    return null
+  }
```
Comment (trollepierre) [inline src/drive/lib/encryption.js:23]: let's update the changelog also 🙏

**Review by trollepierre — COMMENTED** [review]
Thanks for the feature!

File: src/drive/lib/encryption.js (hunk 2 above)
Comment (paultranvan) [reply src/drive/lib/encryption.js:2]: * Ah right, I forgot the existence of this list
* We don't want to enable this feature yet

Comment (trollepierre) [reply src/drive/lib/encryption.js:2]: even for Cozy internal ? (it means the employee of cozy)

File: src/drive/lib/encryption.js (hunk 3 above)
Comment (paultranvan) [reply src/drive/lib/encryption.js:23]: Sorry, I will eventually think of this... Maybe an automatic reminder in the PR would be useful?

Comment (trollepierre) [reply src/drive/lib/encryption.js:23]: https://github.com/cozy/cozy-drive/pull/2541

Comment (trollepierre) [reply src/drive/lib/encryption.js:23]: so bad memory, or so happy with automatisation ^^

**Review by trollepierre — APPROVED** [review]

File: src/drive/lib/encryption.js [hunk 4]
```diff
@@ -17,6 +18,9 @@ export const isEncryptedFile = file => {
 }
 
 export const getEncryptionKeyFromDirId = async (client, dirId) => {
+  if (!flag('drive.enable-encryption')) {
```
[the remark this answers is not in the document]
Comment (paultranvan) [reply src/drive/lib/encryption.js:21]: Just for your pleasure: [70f2ee5](https://github.com/cozy/cozy-drive/pull/2539/commits/70f2ee5b1a963f7bda24a64f9411d28d57c1e751)

**Review by trollepierre — APPROVED** [review]

**Pull request merged**